content/docs-beta/SUSHI/migration/_index.md

```diff
@@ -5,42 +5,102 @@ weight: 30
+
+### 2 - Rearrange Directories
+
+Move the entire contents of **./fsh/ig-data** directory to the top level (do not move the **/ig-data** directory, just its contents). 
+
+{{% alert title="Note" color="primary" %}}
+If after the cleanup, you still have a top-level **./input** directory, then _merge_ the contents of **./fsh/ig-data/input** into that directory (rather than replacing it).
+{{% /alert %}}
+
+Next, move the entire **./fsh** directory into the **./input** directory (resulting in a **./input/fsh** directory).
+
+Finally, delete the **./input/fsh/ig-data** directory (which should be empty).
+
+Your directory structure should now look something like this:
+
+```text
+.
+└── input
+│   ├── fsh
+│   ├── images
+│   └── pagecontent
+└── (other directories such as input-cache, output, temp, template)
+```
+
+### 3 - Move and Rename config.yaml
+
+Move the **./input/fsh/config.yaml** file to the top-level folder, and rename it **sushi-config.yaml**.
+
+### 4 - Run SUSHI and Follow Instructions
+
+You can now run SUSHI 1.0. You may get further instructions in the form of error messages.
+
+One error message concerns the `template` property in **sushi-config.yaml**. If you get that error, follow the instructions to remove that property from **sushi-config.yaml** and create an **./ig.ini** file.
 
 {{% alert title="Warning" color="warning" %}}
-As of the SUSHI 1.0 Beta release, the **ig.ini** file MUST use a template based on `fhir.base.template#current`. Specific template versions (i.e., other than `#current`) are expected to work in the future.  For now, any of the following should work:
+You must specify a template based on `fhir.base.template#current`. Older templates will not work with SUSHI 1.0. Any of the following should work:
+
   * `template = fhir.base.template#current`
   * `template = hl7.base.template#current`
   * `template = hl7.fhir.template#current`
   * `template = hl7.davinci.template#current`
   * `template = hl7.cda.template#current`
 {{% /alert %}}
 
-Update **sushi-config.yaml** to remove the `history` property if necessary. The `history` property is used by SUSHI to generate a **package-list.json** file, and its usage is now deprecated. Similar to `template` above, if SUSHI detects usage of the `history` property to generate a **package-list.json** file, it will log an error message containing the contents of the **package-list.json** that would be generated by the `history` property. Copy these contents and place them in a **package-list.json** file located at **customized-ig/package-list.json**.
+Another error message you might receive concerns the `history` property in **sushi-config.yaml**. If you get that message, remove that property and create the **package-list.json** file as instructed.
+
+When you get a clean build with SUSHI, try running the IG Publisher (use the `_genonce` script). This will verify that all files needed by the IG Publisher are in the right place.
```

Comment:
Is it certain that someone has the `_genonce` script? SUSHI doesn't make that for them anymore.

Reply:
For a migration, we should be able to assume that the user knows how to run the IG Publisher by now (whether via `_genonce` or some other way).  So... I think this is harmless, but we could remove the parenthetical statement entirely or qualify it (e.g., "using the `_genonce` script or whatever method you used in the past").

Reply:
"When you get a clean build with SUSHI, try running the IG Publisher using your customary method (e.g., using the `_genonce` script). This will verify that all files needed by the IG Publisher are in the right place."